src/Owin.Security.OpenIdConnect.Server/OpenIdConnectServerHandler.Endpoints.cs

```diff
@@ -1625,97 +1649,235 @@ internal partial class OpenIdConnectServerHandler : AuthenticationHandler<OpenId
                     RequestType = OpenIdConnectRequestType.AuthenticationRequest
                 };
             }
-            
-            AuthenticationTicket ticket;
-            if (!string.IsNullOrEmpty(request.AccessToken)) {
-                ticket = await ReceiveAccessTokenAsync(request.AccessToken, request);
-            }
-
-            else if (!string.IsNullOrEmpty(request.IdToken)) {
-                ticket = await ReceiveIdentityTokenAsync(request.IdToken, request);
-            }
-
-            else if (!string.IsNullOrEmpty(request.GetRefreshToken())) {
-                ticket = await ReceiveRefreshTokenAsync(request.GetRefreshToken(), request);
-            }
 
-            else {
-                await SendErrorPayloadAsync(new OpenIdConnectMessage {
+            if (string.IsNullOrWhiteSpace(request.Token))
+            {
+                await SendErrorPayloadAsync(new OpenIdConnectMessage
+                {
                     Error = OpenIdConnectConstants.Errors.InvalidRequest,
-                    ErrorDescription = "A malformed validation request has been received: " +
-                        "either an identity token, an access token or a refresh token must be provided."
+                    ErrorDescription = "A malformed introspection request has been received: " +
+                        "a 'token' parameter with an access, refresh, or id token is required."
                 });
 
                 return;
             }
 
-            if (ticket == null) {
-                Options.Logger.WriteError("invalid token");
+            //TODO: Re-examine authentication logic to ensure authentication happens in some way.
+            //TODO: Allow extensibility to skip authentication somehow for access tokens.
+            //      Perhaps in Options.Provider.ValidationEndpoint()?
+            // Attempt to Recieve an Access Token for authentication
+            // See https://tools.ietf.org/html/rfc7662#section-2.1
+            var authTicket = await ReceiveAccessTokenAsync(request.Token, request);
```

Comment:
Very well. As for this PR, it was really just meant for feedback. I'll create another PR with the changes discussed herein.